sdk/documenttranslation/azure-ai-documenttranslation/samples/async_samples/sample_batch_translation_async.py

```diff
@@ -0,0 +1,101 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import os
+import asyncio
+
+
+class BatchTranslationSampleAsync(object):
+
+    async def batch_translation_async():
```

Comment:
Will need `self` if we place in the sample class

```suggestion
    async def batch_translation_async(self):
```